data_portal/viewsets/labmetadata.py

```diff
@@ -66,3 +68,38 @@ def sync(self, request):
         except Exception as e:
             logger.error(e)
             return Response(data={'message': "INTERNAL_SERVER_ERROR"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    @action(detail=False, methods=['get'])
+    def by_aggregate_count(self, request):
+        fields = self.request.query_params.getlist('fields', None)
+
+        if fields is None or not fields:
+            return Response(data={})
+
+        if 'all' in fields:
+            fields.clear()
+            fields.extend(allowed_fields)
+
+        fields = list(set(fields) & set(allowed_fields))  # intersect
+
+        data = {}
+        for f in fields:
+            data.update({
+                str(f): LabMetadata.objects.get_by_aggregate_count(str(f))
+            })
+
+        return Response(data=data)
+
+    @action(detail=False, methods=['get'])
+    def by_cube(self, request):
+        fields = self.request.query_params.getlist('fields', None)
+
+        if fields is None or not fields or len(fields) < 2:
```

Comment:
Wondering why `< 2` instead of ` != 2`? Seems to be more than 2 is also not supported anyhow?

Reply:
> more than 2 is also not supported 

Yup. That's the intention. ditto

> At the mo, supporting 2D to start with